exercises/collatz-conjecture/collatz_conjecture_test.exs

```diff
@@ -0,0 +1,48 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("collatz_conjecture.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule CollatzConjectureTest do
+  use ExUnit.Case
+
+  test "zero steps for one" do
+    assert CollatzConjecture.calc(1) == 0
+  end
+
+  test "zero is an error" do
+    assert_raise ArgumentError, "Only positive numbers are allowed", fn -> CollatzConjecture.calc(0) end
```

Comment:
I wasn't sure if I needed to follow the pattern as in x-common. I put the error case first as forgetting this might lead to an infinite loop later on.